index.html

```diff
@@ -239,12 +239,40 @@ <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
                                      
                                   </div>
                               </div>
-                    <section>
-          
-          
-                     
+                    <section>                 
             
       </div>
+      <div class="font-contact">
+        <section class="contact-form">
+
+            <div class="flex-center">
+                <div><h2 class="title-form">Contact me</h2></div>
+            <div><p class="text-contact"> if you have an application you are interested in developing a feature that you need built a project that need coding .I'd love to help with it
+                </p>
+            </div>
+        
+            <div>
+                <form class="form0"  action="https://formspree.io/f/{form_id}" method="post">
+
+                    <input type="text" class="inp-name" name="name" id="" maxlength="30" placeholder="Massa zoumanigui"  required>
+
+                    <input type="email" class="inp-email" name="name" id="email" placeholder="zoumaniguimass12@gmail.com"  required>
+
+                    <textarea name="user_message" class="inp-text" id="textarea"  placeholder="Write your message here" maxlength="500" required></textarea>
+                                  
```

Comment:
I also notice the following that you can do to improve your project even further.
- Please add `padding-left` to the input elements as indicated by the red arrow in the image below 👇🏾  
- For the textarea element, the padding should be both top and left.
- The `get in touch` button is aligned to the left and not center
- There should be a thick-dark line below the `get in touch` button as observed in the Figma design. Kindly add this to your project.

Your project | Figa Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/222112411-c49a8848-95d2-4fa1-a8e9-da979ebb9557.png) | ![image](https://user-images.githubusercontent.com/84156714/222113102-09c00c46-7c68-47e8-a8d3-73e629ba76fc.png)